src/testUtils/java/org/broadinstitute/hellbender/testutils/VariantContextTestUtils.java

```diff
@@ -2,15 +2,13 @@
 
 import com.google.common.annotations.VisibleForTesting;
```

Comment:
Did you mean to include this file?

Reply:
I think that's where the `@VisibleForTesting` attribute is defined, isn't it?